src/protean/core/entity.py

```diff
@@ -458,7 +469,7 @@ def __init__(self, *template, **kwargs):
         # Now load the remaining fields with a None value, which will fail
         # for required fields
         for field_name, field_obj in self._meta.declared_fields.items():
-            if field_name not in loaded_fields:
+            if field_name not in loaded_fields and not isinstance(field_obj, (HasOne, HasMany)):
```

Comment:
Need a better way to do this

Reply:
Now I can change this to compare against `Association`. Will change that.